src/test/lens.rs

```diff
@@ -46,24 +48,13 @@ fn test_lens_run() {
         ls_server::LsService::handle_message(&mut server),
         ls_server::ServerStateChange::Continue
     );
-    expect_messages(
-        results.clone(),
-        &[
-            ExpectedMessage::new(Some(0)).expect_contains(r#""codeLensProvider":{"resolveProvider":false}"#),
```

Comment:
Should probably check for this message

Reply:
Indeed! Restored other messages as well to try to debug #925